Palashio/libra
# Pushing Restructed Code

---
name: b) New feature
about: Select this if you're adding a new feature to Libra.
labels: feature
---



This pull request closes #issue_number_here .

**- What I did**
Restructured Code as requested

**- How I did it**
Changed file names and locations; used PyCharm change references

**- How to verify it**
Run Tests on my branch to ensure it works (worked on my tests)




- [ ] I updated the docs.

This pull request adds a new feature to Libra. @Palashio, could you please take a look at it?
